### PR TITLE
Separated support policy from licensing

### PR DESCRIPTION
**Pull request by andreasohlund**
@SimonCropp I put the file in /upgrades , better ideas?

@udidahan FYI

### CODE REVIEW

File: nservicebus/licensing/index.md [hunk 1]
```diff
@@ -4,35 +4,9 @@ summary: Outlines the usage, management and restrictions of the NServiceBus lice
 tags: []
 related:
  - nservicebus/upgrades/release-policy
+- nservicebus/upgrades/support-policy
```
Comment (SimonCropp) [inline nservicebus/licensing/index.md]: Missing space

**Comment by SimonCropp** [comment]
Did u scan other pages for links to this content?

**Comment by andreasohlund** [comment]
>Did u scan other pages for links to this content?

Good point, I'll go through what we have

**Comment by andreasohlund** [comment]
@SimonCropp found one link that needed to be changed, all should be ok now

**Pull request merged**